src/mongo_odm/nvp.hpp

```diff
@@ -638,11 +732,11 @@ class nvp_base {
      * @param  sort    An optional argument containing an expression for the $sort modifier.
      * @param  position    An optional argument containing the value of the $position modifier.
      */
-    template <typename U = no_opt_type, typename = std::enable_if_t<is_iterable_not_string_v<U>>,
-              typename Iterable, typename = enable_if_matching_iterable_t<Iterable>,
-              typename Sort = int,
-              typename = std::enable_if_t<details::is_sort_expression_v<Sort> ||
-                                          std::is_same<int, Sort>::value>>
+    template <
+        typename U = no_opt_type, typename = std::enable_if_t<is_iterable_not_string_v<U>>,
```

Comment:
Seems like this change is formatting only? I think clang-format was green before. What happened?